docs/guides/session-management/low-latency-session-management.mdx

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
General remark - heading structure is flat - H2 all across the doc. Maybe you should use H3s too to organize the content better?

Reply:
Added H3 headings